src/common/IOSEnvironmentSetup.ts

```diff
@@ -4,22 +4,14 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import { Logger, Messages } from '@salesforce/core';
-import childProcess from 'child_process';
+import { Logger } from '@salesforce/core';
 import util from 'util';
 import iOSConfig from '../config/iosconfig.json';
+import { CommonUtils } from './CommonUtils';
 import { IOSUtils } from './IOSUtils';
 import { BaseSetup } from './Requirements';
 
-const exec = util.promisify(childProcess.exec);
-
 export class IOSEnvironmentSetup extends BaseSetup {
-    public static executeCommand(
```

Comment:
Again, there's no point in having this function here when all it does is call into another function.